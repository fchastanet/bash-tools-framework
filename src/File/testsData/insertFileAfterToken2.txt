stuff before
# FUNCTIONS
stuff after
# FUNCTIONS
stuff again after
