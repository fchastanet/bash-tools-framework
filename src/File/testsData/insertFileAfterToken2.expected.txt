stuff before
# FUNCTIONS
insertFileAfterToken injected
stuff after
# FUNCTIONS
stuff again after
